bq-client.js

```diff
@@ -20,7 +20,7 @@ module.exports = (projectName, dataSetName, refreshUrl)=>{
 
   var mod = {
     refreshToken,
-    insert(tableName, data, nowDate) {
+    insert(tableName, data, templateSuffix, nowDate) {
```

Comment:
@tejohnso I don't know I thought it differently. Makes sense. Changes made. cheers